src/components/interactive-map.js

```diff
@@ -27,6 +29,12 @@ export default class InteractiveMap extends PureComponent {
     autobind(this);
   }
 
+  getChildContext() {
```

Comment:
If I understand it correctly, the downside of the context approach, compared to child prop injection, is that every component that wraps `StaticMap` needs to define it in this precise way for the `Marker`s to work? 

That seems to be a weakness as this InteractiveMap would need to be defined by apps that want to build their own event handling, which arguably is a key thing we are enabling in v3.0.0

Thoughts?